.github/workflows/test_release_packages.yml

```diff
@@ -0,0 +1,118 @@
+name: Test release packages
+
+on:
+  workflow_run:
+    workflows: ["Build portable Linux packages"]
+    types:
+      - completed
+
+permissions:
+  contents: write
```

Comment:
Do all jobs here need write? If not, I would set it to read here and only grant additional permissions where needed.